src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -106,9 +115,8 @@ public SortedSet(IEnumerable<T> collection, IComparer<T> comparer)
                 }
                 return;
             }
-            
-            int count;
-            T[] elements = EnumerableHelpers.ToArray(collection, out count);
+
+            T[] elements = EnumerableHelpers.ToArray(collection, out int count);
```

Comment:
@ViktorHofer That's a shame, I really like skipping the declaration. I'll follow your suggestion, though.